proto/messages.proto

```diff
@@ -206,7 +207,8 @@ message Asset {
 // Actual message is encrypted with AES and sent as additional data
 message External {
   required bytes otr_key = 1;
-  optional bytes sha256 = 2;      // sha256 of ciphertext
+  optional bytes sha256 = 2;      // sha256 of ciphertext, obsolete but required for backward compatibility
```

Comment:
During the transition period there will be clients which don't know nothing about the possibility of using another algorithms, and clients which know about it, but still use the old one. I think it's better to make it clear in the code which one is which one, even if it doesn't change much. On the other hand, it doesn't require much additional work and it will be easier to read for developers. After the transition period we will delete it anyway.